### PR TITLE
add fuzz test for store package

### PR DESCRIPTION
**Pull request by AdamKorcz**
Adds the fuzzer from https://github.com/google/oss-fuzz/pull/12193

### CODE REVIEW

File: pkg/store/fuzz_test.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+func FuzzLoadYAMLByFilePath(f *testing.F) {
+	f.Fuzz(func(t *testing.T, fileContents []byte) {
+		err := os.WriteFile("yaml_file.yml", fileContents, 0o600)
+		if err != nil {
+			t.Fatal(err)
+		}
+		defer os.Remove("yaml_file.yml")
+
+		_, _ = LoadYAMLByFilePath("yaml_file.yml")
```
Comment (jandubois) [inline pkg/store/fuzz_test.go:16]: Isn't the fuzzing function missing an `// Output:` comment?

File: pkg/store/fuzz_test.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package store
+
+import (
+	"os"
+	"testing"
+)
+
+func FuzzLoadYAMLByFilePath(f *testing.F) {
+	f.Fuzz(func(t *testing.T, fileContents []byte) {
+		err := os.WriteFile("yaml_file.yml", fileContents, 0o600)
```
[the remark this answers is not in the document]
Comment (afbjorklund) [reply pkg/store/fuzz_test.go:10]: I think you want to use `testing.TempDir`

[the remark this answers is not in the document]
Comment (AdamKorcz) [reply pkg/store/fuzz_test.go]: https://github.com/lima-vm/lima/pull/2478/commits/a173705c9817cc0da6698678a1cb7eaf906f935d

File: pkg/store/fuzz_test.go (hunk 1 above)
Comment (AdamKorcz) [reply pkg/store/fuzz_test.go]: I am not familiar with `// Output:`. Could you make a code suggestion? Or do you have a link to documentation for that?

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Pull request merged**